src/all.jsdom.test.js

```diff
@@ -32,5 +32,14 @@ describe('Init All', () => {
       HMRCFrontend.initAll();
       expect(BackLinkHelper.prototype.init).toHaveBeenCalledTimes(2);
     });
+
+    it('should initialise correctly configured elements found on the page', () => {
+      addBackLinkToPage();
+      const brokenTimeoutDialog = document.createElement('meta');
+      brokenTimeoutDialog.setAttribute('content', 'hmrc-timeout-dialog'); // broken configuration of timeout-dialog will error
```

Comment:
sorry, I think we will need the other attributes - or at least `meta[name="hmrc-timeout-dialog"` otherwise it won't try to init timeout dialog and test would pass because no error during initialisation

Reply:
could maybe spy on the console.error during test to check that got error logged to show it attempted to init timeout dialog